code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -353,22 +353,30 @@
 			if(user.getStaminaLoss() > 50)
 				to_chat(user, span_warning("You are too tired to work!"))
 				return
+			var/list/choose_options = list(
+				"Grind" = image(icon = 'icons/obj/objects.dmi', icon_state = "ash"), 
+				"Juice" = image(icon = 'icons/obj/drinks.dmi', icon_state = "glass_orange") 
```

Comment:
```suggestion
				"Juice" = image(icon = 'icons/hud/radial.dmi', icon_state = "radial_grind")

Reply:
I get the idea, but I'll do it tomorrow.
Also, your in your sugg you've placed the grind.dmi in juice and vice versa

Reply:
OOPS!

Reply:
@Microvolnovka19 They should be able to one click now for your convenience.